Cargo.toml

```diff
@@ -10,10 +10,14 @@ homepage = "https://github.com/multiformats/rust-multibase"
 repository = "https://github.com/multiformats/rust-multibase"
 keywords = ["ipld", "ipfs", "multihash", "multibase", "cid"]
 
+[features]
+default = ["std"]
+std = []
+
 [dependencies]
-base-x = "0.2"
-data-encoding = "2.2"
-data-encoding-macro = "0.1.8"
+base-x = { git = "https://github.com/whalelephant/base-x-rs", branch = "no_std", default-features = false }
```

Comment:
Before this PR can be merged, we need to have that on crates.io. I pinged on the [upstream PR](https://github.com/OrKoN/base-x-rs/pull/14) as I really hope we don't need to fork.

Reply:
We don't need a fork! That PR was merged upstream and got released!